<!DOCTYPE html>
<html lang="en" class="no-js">

  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Immaterial Demo -- failed to build</title>

    <link rel="stylesheet" href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css">
  </head>

  <body class="bg-red-50 antialiased max-w-4xl mx-auto p-4 my-2">
    <h1 class="text-red-700 text-5xl text-center">
      Build Failed!
    </h1>
    <div class="mt-2">
      <div class="text-gray-700 text-center font-semibold">17 Mar 2025 01:09:22 UTC</div>
    </div>
    <div class="mt-4">
      <div class="text-red-700 bg-red-100 border border-red-200 p-4 rounded text-center font-semibold">
        Unless all site previews are broken, this is a problem with the
        <a class="underline"
          href="https://pypi.org/project/sphinx-immaterial/"><code>sphinx-immaterial</code></a>
        package.
      </div>
    </div>
    <div class="mt-4">
      <div class="text-gray-700 text-center font-semibold">Configuration</div>
      <pre class="bg-gray-50 border my-2 shadow rounded p-2 overflow-auto">{&#39;html_theme&#39;: &#39;sphinx_immaterial&#39;}</pre>
    </div>
    <div class="mt-4">
      <div class="text-gray-700 text-center font-semibold">Output</div>
      <pre class="bg-gray-50 border my-2 shadow rounded p-2 overflow-auto">======================================== stdout ========================================
[01mRunning Sphinx v8.2.3[39;49;00m
[01mloading translations [en]... [39;49;00mlocale_dir /home/runner/work/sphinx-themes.org/sphinx-themes.org/sample-docs/locales/en/LC_MESSAGES does not exist
locale_dir /home/runner/work/sphinx-themes.org/sphinx-themes.org/sample-docs/locales/en/LC_MESSAGES does not exist
done
[01mmaking output directory... [39;49;00mdone
locale_dir /home/runner/work/sphinx-themes.org/sphinx-themes.org/sample-docs/locales/en/LC_MESSAGES does not exist
loading intersphinx inventory &#39;sphinx&#39; from https://www.sphinx-doc.org/objects.inv ...
intersphinx inventory has moved: https://www.sphinx-doc.org/objects.inv -&gt; https://www.sphinx-doc.org/en/master/objects.inv

======================================== stderr ========================================

[91mExtension error (sphinx_immaterial.google_fonts)![39;49;00m

Versions
========

* Platform:         linux; (Linux-6.8.0-1021-azure-x86_64-with-glibc2.39)
* Python version:   3.12.3 (CPython)
* Sphinx version:   8.2.3
* Docutils version: 0.21.2
* Jinja2 version:   3.1.6
* Pygments version: 2.19.1

Last Messages
=============

None.

Loaded Extensions
=================

None.

Traceback
=========

    Traceback (most recent call last):
      File &#34;/home/runner/work/sphinx-themes.org/sphinx-themes.org/.nox/publish/lib/python3.12/site-packages/sphinx/events.py&#34;, line 404, in emit
        results.append(listener.handler(self.app, *args))
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File &#34;/home/runner/work/sphinx-themes.org/sphinx-themes.org/.nox/publish/lib/python3.12/site-packages/sphinx_immaterial/google_fonts.py&#34;, line 208, in _builder_inited
        font_options = app.config[&#34;html_theme_options&#34;][&#34;font&#34;]
                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^
    KeyError: &#39;font&#39;
    
    The above exception was the direct cause of the following exception:
    
    Traceback (most recent call last):
      File &#34;/home/runner/work/sphinx-themes.org/sphinx-themes.org/.nox/publish/lib/python3.12/site-packages/sphinx/cmd/build.py&#34;, line 414, in build_main
        app = Sphinx(
              ^^^^^^^
      File &#34;/home/runner/work/sphinx-themes.org/sphinx-themes.org/.nox/publish/lib/python3.12/site-packages/sphinx/application.py&#34;, line 332, in __init__
        self._init_builder()
      File &#34;/home/runner/work/sphinx-themes.org/sphinx-themes.org/.nox/publish/lib/python3.12/site-packages/sphinx/application.py&#34;, line 414, in _init_builder
        self.events.emit(&#39;builder-inited&#39;)
      File &#34;/home/runner/work/sphinx-themes.org/sphinx-themes.org/.nox/publish/lib/python3.12/site-packages/sphinx/events.py&#34;, line 415, in emit
        raise ExtensionError(
    sphinx.errors.ExtensionError: Handler &lt;function _builder_inited at 0x7fc837de9260&gt; for event &#39;builder-inited&#39; threw an exception (exception: &#39;font&#39;)


The full traceback has been saved in:
/tmp/sphinx-err-ap85oio_.log

To report this error to the developers, please open an issue at &lt;https://github.com/sphinx-doc/sphinx/issues/&gt;. Thanks!
Please also report this if it was a user error, so that a better error message can be provided next time.
</pre>
    </div>
  </body>

</html>